pkg/kubelet/events/event.go

```diff
@@ -26,6 +26,7 @@ const (
 	PreemptContainer        = "Preempting"
 	BackOffStartContainer   = "BackOff"
 	ExceededGracePeriod     = "ExceededGracePeriod"
+	SysctlForbidden         = "SysctlForbidden"
```

Comment:
@alculquicondor , do you mind giving a short explanation of why we need to move around `pkg/kubelet` constants in order to support removing dependencies in `tests/e2e`? It looks like the change to stop depending on `pkg/kubelet/sysctl.go` is somewhat distinct? I'm not saying its a good or bad change, I'm just having a little trouble seeing how its related :)

Reply:
`pkg/kubelet/sysctl` depends on `pkg/kubelet/lifecycle` which depends on `pkg/scheduler/framework`.

Reply:
the dependency chain goes like this:

`pkg/kubelet/sysctl` -> `pkg/kubelet/lifecycle` -> `pkg/scheduler/framework/...`

Reply:
Ahh I gotcha - mind including that information in the pull request or commit message?

Reply:
lol, thanks @andrewsykim

Reply:
Updated the description